libsolidity/codegen/CompilerUtils.cpp

```diff
@@ -1030,7 +1045,6 @@ void CompilerUtils::convertType(
 			}
 			case DataLocation::CallData:
 			{
-				solUnimplementedAssert(!typeOnStack.isDynamicallyEncoded(), "");
```

Comment:
We should let the following `abiDecode()` decide whether dynamically encoded type is allowed in calldata struct. (If ABIEncoderV2 is enabled, it should be allowed)